packages/ia-topnav/src/styles/dropdown-menu.js

```diff
@@ -78,6 +78,16 @@ export default css`
     font-weight: bold;
   }
 
+  a.mobile-upload {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+  }  
+  a.mobile-upload svg {
+    fill: #fff;
```

Comment:
ok. q: is there style based css var for this?

Reply:
changed with var(--white) variable. thanks!